monitor/service.go

```diff
@@ -200,6 +196,12 @@ func (m *Monitor) Statistics(tags map[string]string) ([]*Statistic, error) {
 					}
 					statistic.Values[kv.Key] = f
 				})
+			case "references":
+				s := subKV.Value.(*expvar.Int).String()
+				count, _ := strconv.ParseInt(s, 10, 32)
+				if count < 2 {
+					influxdb.DeleteStatistics(kv.Key)
```

Comment:
I'm confused about why we delete the stats here. It seems to be related to setting the initial references to 2, but I'm questioning if it's necessary to track the reference for the monitor.